packages/docs-site/src/content/docs/network-reference/network-configuration.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Network configuration
+description: The network configuration page describes many of the configuration details of the network.
+---
+
+import {Aside} from '@astrojs/starlight/components';
+
+## Cooldown Window, Proving Window and Bonds
+
+This segment lists the cooldown windows and proving windows for every tier of proof in the Taiko protocol.
+
+The cooldown window describes how long after the proof has been provided that the block is verified.
+
+The proving window describes how long a prover has to provide a proof for a block after they have been assigned the block (i.e. a block has been proposed with them as the assigned prover).
+
+You can find all the corresponding values below in the above link too!
+
+| Tier              | Cooldown Window | Proving Window | Bond      |
+| ----------------- | --------------- | -------------- | --------- |
+| Optimistic        | 1440 minutes    | 255 minutes    | 75 TAIKO  |
+| SGX               | 240 minutes     | 300 minutes    | 150 TAIKO |
+| ZK                | 240 minutes     | 420 minutes    | 225 TAIKO |
+| SGX & ZK          | 240 minutes     | 420 minutes    | 225 TAIKO |
+| Guardian Minority | 240 minutes     | 240 minutes    | 225 TAIKO |
+| Guardian          | 240 minutes     | 240 minutes    | 0 TAIKO   |
+
+## Tier configuration (Mainnet)
+
+You can view the full `MainnetTierRouter` tier configuration [here](https://github.com/taikoxyz/taiko-mono/blob/main/packages/protocol/contracts/layer1/mainnet/rollup/MainnetTierRouter.sol):
```

Comment:
Let's use this link? https://etherscan.io/address/0x8f1C1D58C858e9a9eeCc587d7D51AECfd16b5542#code , code in github link will be updated after merging another PR, but we won't deploy it these days.